components/CodeBlock/CodeBlock.tsx

```diff
@@ -1,7 +1,7 @@
 import { Prism as SyntaxHighlighter } from "react-syntax-highlighter"
 import { dracula } from "react-syntax-highlighter/dist/cjs/styles/prism"
 
-type TCodeLanguage =
+type TcodeLanguage =
```

Comment:
Although this technically solves the issue I don't think this is a good pattern to follow. It is supposed to be named `TCodeLanguage`. I think we need to update the config file instead to allow types to follow this naming convention.

```
{
"selector": "type",
"format": ["PascalCase"]
},
```

Reply:
I see Cristian has already specified the format for `typeLikes` in eslintrc.json file, how about I update the convention to also include PascalCase like this
```js
{
        "selector": "typeLike",
        "format": ["StrictPascalCase", "PascalCase"]				
},
```
It does solves the issue while keeping `TCodeLanguage` as it is

Reply:
Sure!